src/components/payment/buy-form.tsx

```diff
@@ -45,6 +45,22 @@ const _cardSchema = z.object({
   }),
 });
 
+export const aditionalAddressSchema = z.object({
+  adtionalCity: z.string({
```

Comment:
Same typo as the previous PR.
```suggestion
  additionalCity: z.string({
```

Reply:
Yes, this is weird, because the code merged by the previous PR is missing here 🤔 Maybe is out of date with main?